### ISSUE TITLE
TypeError: edge.initializeClrFunc is not a function

### ISSUE THREAD

**Issue by codeyu** [agracio/edge-js-quick-start#1]
I fork this project, run and get a error as title.
OS: macOS 10.13.2 
Node.js: 8.8.1
NETCore: 2.0.1-servicing-006924
mono: 5.2.0.215

exception stack:
```
edge-js-quick-start git:(master) node main.js
CoreClrEmbedding::Initialize - Failed to initialize CoreCLR, HRESULT: 0x80070057
/github/edge-js-quick-start/node_modules/edge-js/lib/edge.js:178
    return edge.initializeClrFunc(options);
                ^

TypeError: edge.initializeClrFunc is not a function
    at Object.exports.func (/github/edge-js-quick-start/node_modules/edge-js/lib/edge.js:178:17)
    at Object.<anonymous> (/github/edge-js-quick-start/main.js:12:34)
    at Module._compile (module.js:612:30)
    at Object.Module._extensions..js (module.js:623:10)
    at Module.load (module.js:531:32)
    at tryModuleLoad (module.js:494:12)
    at Function.Module._load (module.js:486:3)
    at Function.Module.runMain (module.js:653:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
```

**Comment by zhu1531** [comment]
me too

```
node main.js

module.js:597
  return process.dlopen(module, path._makeLong(filename));
                 ^
Error: Call to coreclr_create_delegate() for G failed with a return code of 0x80070002.
    at Error (native)
    at Object.Module._extensions..node (module.js:597:18)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\zr\Desktop\edge-js-quick-start-master\node_modules\edge-js\lib\edge.js:53:8)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
```

**Comment by gianlazz** [comment]
I am also having this issue
macOS 10.13.3
node.js v8.9.4
NETCore 2.1.101

**Comment by agracio** [comment]
Was very busy lately, will take a look at macOS implementation issues in the next few days.

**Comment by agracio** [comment]
Issue fixed in new `edge-js` release. Quick start have been updated, and now works on Windows/macOS/Linux. Either do a fresh repo clone and follow the setup steps or update your repo, perform `npm install` and rebuild .NET solution.

**Comment by voxsoftware** [comment]
Same problem 
node 10 
ubuntu 18.04 x64
net core 2.2.105

**Comment by Pphoenix666** [comment]
Hello, I cloned it, did npm install, then build the sln project and got :
 Uncaught TypeError: edge.initializeClrFunc is not a function
    at Object.exports.func (C:\A_Test\node_modules\electron-edge-js\lib\edge.js:187)

I have NodeJS 12.4.0
Electron 5.0.3
Electron-edge-js 12.4.0

Tried to download it rather than clone it, then building the cs part with the command, still didn't work.
Tried changing the paths to full direct string paths rather than variables.
Read the others posts and tried all the propositions.
Still didn't found how to fix it :'(
